sources/be/vlg/bosa-region-flanders-nl.json

```diff
@@ -0,0 +1,42 @@
+{
+    "data": "https://opendata.bosa.be/download/best/openaddress-bevlg.zip",
+    "protocol": "http",
+    "coverage": {
+         "ISO 3166": {
+             "alpha2": "BE-VLG",
+             "state": "Flemish Region",
+             "country": "Belgium"
+         },
+         "country": "be",
+         "state": "flemish region"
+    },
+    "conform": {
+        "format": "csv",
+        "srs": "EPSG:4326",
+        "encoding": "utf-8",
+        "accuracy": 3,
+        "number": "house_number",
+        "street": "streetname_nl",
+        "city":  "muncipality_name_nl",
+        "postcode": "postcode",
+        "id": {
+            "function": "join",
+            "fields": ["region_code","address_id"],
+            "separator": ":"
+        },
+        "unit": "box_number",
+        "lon": "EPSG:4326_lon",
+        "lat": "EPSG:4326_lat"
+    },
+    "compression": "zip",
+    "language": "nl",
+    "website": "https://opendata.bosa.be",
+    "license": {
+        "url":"https://overheid.vlaanderen.be/sites/default/files/documenten/ict-egov/licenties/hergebruik/modellicentie_gratis_hergebruik_v1_0.html"
```

Comment:
again just using Google Translate here, but it appears to require some kind of attribution? If so we should set atttribution: true and an "attribution name"

Reply:
the license does indeed require attribution, I've added the attributes